src/server/apis/user.ts

```diff
@@ -28,8 +28,40 @@ async function handlerUserRequest(
   });
 }
 
+export async function handleRemoveRepo(
+    request: express.Request, userRecord: UserRecord): Promise<APIResponse> {
+  const owner = request.body.owner;
+  const name = request.body.name;
+
+  if (!owner || !name) {
+    return responseHelper.error(
+        'invalid-request', 'Missing owner/name from request');
+  }
+
+  const oldRepos = userRecord.repos;
+  const newRepos = [];
+  if (!oldRepos) {
+    return responseHelper.error('no-repos', 'No repos found on user');
+  }
+
+  for (const repo of oldRepos) {
+    if (repo.owner !== owner || repo.name !== name) {
+      newRepos.push(repo);
```

Comment:
Could you change this to splice out the one repo from the array and break when it's found?

Reply:
Done.